apc_lock.c

```diff
@@ -281,8 +281,12 @@ PHP_APCU_API void apc_lock_cleanup() {
 
 PHP_APCU_API zend_bool apc_lock_create(apc_lock_t *lock) {
 	char lock_path[] = "/tmp/.apc.XXXXXX";
+#if defined(HAVE_MKSTEMP)
```

Comment:
As we already have unconditional usage in https://github.com/krakjoe/apcu/blob/master/apc_mmap.c#L111, we can use unconditionally here as well.